app/assets/javascripts/transcripts.js

```diff
@@ -65,6 +65,32 @@ $(document).ready(function(){
       new ImageCrop
     }, 100 );
   };
+
+  $('#select-all').click(function () {
+    $('.check').prop('checked', $(this).prop('checked'));
+  });
+
+  $('.item-list-headings').on('click', '.update-multiple', function(e) {
```

Comment:
is it possible to have these calls on an id or data attribute instead of class? it's a bit more robust that way